extensions/amp-analytics/0.1/test/test-config.js

```diff
@@ -882,6 +883,86 @@ describes.realWin('AnalyticsConfig', {amp: false}, env => {
     });
   });
 
+  describe('warning message', () => {
+    it('shows the warning', () => {
+      ANALYTICS_CONFIG['-test-venfor'] = {
+        'requestOrigin': 'https://example.com',
+        'requests': {'test1': '/test1', 'test2': '/test1/test2'},
+        'warningMessage': 'I am a warning',
+      };
+
+      const element = getAnalyticsTag({}, {'type': '-test-venfor'});
+      const usrObj = user();
+      const spy = sandbox.spy(usrObj, 'warn');
+
+      return new AnalyticsConfig(element).loadConfig().then(config => {
+        expect(spy).callCount(1);
+        expect(config['warningMessage']).to.be.undefined;
+      });
+    });
+
+    it('does not always show the warning', () => {
```

Comment:
Hmmm I don't about this one. Seems to only cover a if statement... I'd prefer to have this one deleted.